absorption_modalaSean.py

```diff
@@ -65,55 +65,44 @@
 
 
 def smooth(norm_flux, box_size):       
-    #box = np.ones(box_size)/box_size
-    #y_smooth = convolve(norm_flux, box, mode='same')
     y_smooth = signal.savgol_filter(norm_flux,box_size,2)  #linear
     return y_smooth
 
 
 fig= plt.figure()
 
-#For IDE run, locate the file
-config_file = "sorted_norm.csv" 
-
-#For command line, activate this config_file
-#config_file = sys.argv[1]
+CONFIG_FILE = sys.argv[1] if len(sys.argv) > 1 else "sorted_norm.csv"
 
 spectra_list, redshift_value_list, snr_value_list = [], [], []
-
-
-for line in open(config_file, 'r'):
+for line in open(CONFIG_FILE, 'r'):
 	each_row_in_file = line.split(",")
 	spectra_list.append(each_row_in_file[0])
 	redshift_value_list.append(np.float(each_row_in_file[1]))
 	snr_value_list.append(np.float(each_row_in_file[2]))
 
 
-file_index = 0
-# Start of loop for Spectra
-for spectrum_file_name, redshift_value, snr_value  in zip (spectra_list, redshift_value_list, snr_value_list):   
+file_count = 0
+for spectrum_file_name, redshift_value, snr_value in zip (spectra_list, redshift_value_list, snr_value_list):   
     
     #########################################################
     ##If you dont have all files, point out stopping variable
-    ##In this case, I will stop it after 5th file
-    if file_index == 10:
+    ##In this case, I will stop it after 10th file
+    if file_count == 10:
```

Comment:
10 should be a variable with a meaningful name that you could set somewhere on top of the file